examples/authentication/src/auth/demo-auth/TokenInBrowserStorageService.ts

```diff
@@ -0,0 +1,41 @@
+import {AuthTokenActions, Token} from '@brightinteractive/bright-js-framework/plugins/auth/auth'
+import {authTokenActions} from '@brightinteractive/bright-js-framework/plugins/auth'
+import {browserStorage, BrowserStorageSystem} from '@brightinteractive/bright-js-framework/plugins/browserstorage'
+import {Service} from '@brightinteractive/bright-js-framework'
+
+export default class TokenInBrowserStorageService extends Service<any> {
+
+    ID_TOKEN_KEY = 'ap_auth0_id_token'
```

Comment:
Probably change this to not have auth0 in the string

Reply:
Done.